src/py/flwr/common/serde_test.py

```diff
@@ -307,3 +381,59 @@ def test_recordset_serialization_deserialization() -> None:
     # Assert
     assert isinstance(proto, ProtoRecordSet)
     assert original == deserialized
+
+
+def test_message_to_and_from_task_ins() -> None:
```

Comment:
In line with how we did it in one of the other recent PRs:
```suggestion
def test_message_to_and_from_taskins() -> None:
```